src/libs/dbHandler.go

```diff
@@ -616,3 +619,45 @@ func getSysSummarySQL(db *sql.DB, dbName string, filterOptions types.SystemSumma
 
 	return resSummary, err
 }
+
+// ==================================== //
+// == Purge Old DB Entries Cron Job ==  //
+// ==================================== //
+var (
+	CfgDB          types.ConfigDB
+	PurgeDBCronJob *cron.Cron
+	PurgeDBMap     types.ConfigPurgeOldDBEntries
+)
+
+func InitPurgeOldDBEntries() {
+	log = logger.GetInstance()
+	CfgDB = cfg.GetCfgDB()
+
+	if cfg.GetCfgPurgeOldDBEntriesEnable() {
+
+		PurgeDBCronJob = cron.New()
+		err := PurgeDBCronJob.AddFunc(cfg.GetCfgPurgeOldDBEntriesCronJobTime(), PurgeOldDBEntries) // time interval
+		if err != nil {
+			log.Error().Msg(err.Error())
+			return
+		}
+		PurgeDBCronJob.Start()
+		log.Info().Msg("Purging Old DB Entries cron job started")
+	}
+}
+
+// Checking which type of Database
+func PurgeOldDBEntriesCronJob(cfg types.ConfigDB) {
```

Comment:
This is a cron job CB function and does not have an input. 
You need to read the config from config directly and process the same.

Reply:
Resolved.